charms/kfp-viewer/src/components/pebble_component.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+import logging
+from typing import Dict
+
+from charmed_kubeflow_chisme.components.pebble_component import PebbleServiceComponent
+from ops.pebble import Layer
+
+logger = logging.getLogger(__name__)
+
+
+class PebbleServiceContainerComponent(PebbleServiceComponent):
```

Comment:
Should we have container name in class name? It has one-to-one mapping to only one container.
For this charm it does not really matter, but we want to establish pattern. For example. multi-container charms will have two different PebbleSeriviceContainerComponents.